Description: Prevent usage of fonts.gstatic.com and fonts.googleapis.com to prevent Lintian
 warning about potential privacy-breach-generic issues.
Author: Hugh McMaster
Last-Update: 2020-06-20

diff --git a/docs/reference/404.html b/docs/reference/404.html
index 4f0f50e..dae2a5d 100644
--- a/docs/reference/404.html
+++ b/docs/reference/404.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-auto_hinter.html b/docs/reference/ft2-auto_hinter.html
index 3b3844f..6a5c598 100644
--- a/docs/reference/ft2-auto_hinter.html
+++ b/docs/reference/ft2-auto_hinter.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-base_interface.html b/docs/reference/ft2-base_interface.html
index b0fc2e0..d5f20f8 100644
--- a/docs/reference/ft2-base_interface.html
+++ b/docs/reference/ft2-base_interface.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-basic_types.html b/docs/reference/ft2-basic_types.html
index 478a4d3..080678a 100644
--- a/docs/reference/ft2-basic_types.html
+++ b/docs/reference/ft2-basic_types.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-bdf_fonts.html b/docs/reference/ft2-bdf_fonts.html
index 5ac44a5..178935f 100644
--- a/docs/reference/ft2-bdf_fonts.html
+++ b/docs/reference/ft2-bdf_fonts.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-bitmap_handling.html b/docs/reference/ft2-bitmap_handling.html
index 6b85128..291e0d7 100644
--- a/docs/reference/ft2-bitmap_handling.html
+++ b/docs/reference/ft2-bitmap_handling.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-bzip2.html b/docs/reference/ft2-bzip2.html
index 9cb1c49..4223e3b 100644
--- a/docs/reference/ft2-bzip2.html
+++ b/docs/reference/ft2-bzip2.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-cache_subsystem.html b/docs/reference/ft2-cache_subsystem.html
index 040447a..7e7bd7b 100644
--- a/docs/reference/ft2-cache_subsystem.html
+++ b/docs/reference/ft2-cache_subsystem.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-cff_driver.html b/docs/reference/ft2-cff_driver.html
index 01c3548..f561506 100644
--- a/docs/reference/ft2-cff_driver.html
+++ b/docs/reference/ft2-cff_driver.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-cid_fonts.html b/docs/reference/ft2-cid_fonts.html
index 6673c2e..bc4d5cc 100644
--- a/docs/reference/ft2-cid_fonts.html
+++ b/docs/reference/ft2-cid_fonts.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-color_management.html b/docs/reference/ft2-color_management.html
index b219456..ba4d061 100644
--- a/docs/reference/ft2-color_management.html
+++ b/docs/reference/ft2-color_management.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-computations.html b/docs/reference/ft2-computations.html
index e499955..0ba8892 100644
--- a/docs/reference/ft2-computations.html
+++ b/docs/reference/ft2-computations.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-error_code_values.html b/docs/reference/ft2-error_code_values.html
index a9535e4..140b409 100644
--- a/docs/reference/ft2-error_code_values.html
+++ b/docs/reference/ft2-error_code_values.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-error_enumerations.html b/docs/reference/ft2-error_enumerations.html
index fdb024a..b11f8b1 100644
--- a/docs/reference/ft2-error_enumerations.html
+++ b/docs/reference/ft2-error_enumerations.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-font_formats.html b/docs/reference/ft2-font_formats.html
index 2ac6bc7..4863173 100644
--- a/docs/reference/ft2-font_formats.html
+++ b/docs/reference/ft2-font_formats.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-gasp_table.html b/docs/reference/ft2-gasp_table.html
index 8f483c7..b32ea90 100644
--- a/docs/reference/ft2-gasp_table.html
+++ b/docs/reference/ft2-gasp_table.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-glyph_management.html b/docs/reference/ft2-glyph_management.html
index 1342f20..53e6fa1 100644
--- a/docs/reference/ft2-glyph_management.html
+++ b/docs/reference/ft2-glyph_management.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-glyph_stroker.html b/docs/reference/ft2-glyph_stroker.html
index 907d56c..aec99e0 100644
--- a/docs/reference/ft2-glyph_stroker.html
+++ b/docs/reference/ft2-glyph_stroker.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-glyph_variants.html b/docs/reference/ft2-glyph_variants.html
index 9e3d67e..a206714 100644
--- a/docs/reference/ft2-glyph_variants.html
+++ b/docs/reference/ft2-glyph_variants.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-gx_validation.html b/docs/reference/ft2-gx_validation.html
index 32cba84..297669a 100644
--- a/docs/reference/ft2-gx_validation.html
+++ b/docs/reference/ft2-gx_validation.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-gzip.html b/docs/reference/ft2-gzip.html
index 5782d9a..adfb085 100644
--- a/docs/reference/ft2-gzip.html
+++ b/docs/reference/ft2-gzip.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-header_file_macros.html b/docs/reference/ft2-header_file_macros.html
index 9667487..03c1e3d 100644
--- a/docs/reference/ft2-header_file_macros.html
+++ b/docs/reference/ft2-header_file_macros.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-header_inclusion.html b/docs/reference/ft2-header_inclusion.html
index 44a4caf..5c0684e 100644
--- a/docs/reference/ft2-header_inclusion.html
+++ b/docs/reference/ft2-header_inclusion.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-incremental.html b/docs/reference/ft2-incremental.html
index c7beef8..278938e 100644
--- a/docs/reference/ft2-incremental.html
+++ b/docs/reference/ft2-incremental.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-index.html b/docs/reference/ft2-index.html
index ebd0f06..b6c0657 100644
--- a/docs/reference/ft2-index.html
+++ b/docs/reference/ft2-index.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-layer_management.html b/docs/reference/ft2-layer_management.html
index cfba13e..45b32f5 100644
--- a/docs/reference/ft2-layer_management.html
+++ b/docs/reference/ft2-layer_management.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-lcd_rendering.html b/docs/reference/ft2-lcd_rendering.html
index 6eb75f3..b462d3d 100644
--- a/docs/reference/ft2-lcd_rendering.html
+++ b/docs/reference/ft2-lcd_rendering.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-list_processing.html b/docs/reference/ft2-list_processing.html
index 5262bdb..dc9fadf 100644
--- a/docs/reference/ft2-list_processing.html
+++ b/docs/reference/ft2-list_processing.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-lzw.html b/docs/reference/ft2-lzw.html
index 0bce31c..3470961 100644
--- a/docs/reference/ft2-lzw.html
+++ b/docs/reference/ft2-lzw.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-mac_specific.html b/docs/reference/ft2-mac_specific.html
index 86f25b3..2cb494b 100644
--- a/docs/reference/ft2-mac_specific.html
+++ b/docs/reference/ft2-mac_specific.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-module_management.html b/docs/reference/ft2-module_management.html
index a16364b..cb99aca 100644
--- a/docs/reference/ft2-module_management.html
+++ b/docs/reference/ft2-module_management.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-multiple_masters.html b/docs/reference/ft2-multiple_masters.html
index f02e0c9..7b00cf9 100644
--- a/docs/reference/ft2-multiple_masters.html
+++ b/docs/reference/ft2-multiple_masters.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-ot_validation.html b/docs/reference/ft2-ot_validation.html
index 3fd6499..ccf19a0 100644
--- a/docs/reference/ft2-ot_validation.html
+++ b/docs/reference/ft2-ot_validation.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-outline_processing.html b/docs/reference/ft2-outline_processing.html
index 9ad8482..3f9aedd 100644
--- a/docs/reference/ft2-outline_processing.html
+++ b/docs/reference/ft2-outline_processing.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-parameter_tags.html b/docs/reference/ft2-parameter_tags.html
index 1fb94a5..62f6bc5 100644
--- a/docs/reference/ft2-parameter_tags.html
+++ b/docs/reference/ft2-parameter_tags.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-pcf_driver.html b/docs/reference/ft2-pcf_driver.html
index 75a32f9..4e26e27 100644
--- a/docs/reference/ft2-pcf_driver.html
+++ b/docs/reference/ft2-pcf_driver.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-pfr_fonts.html b/docs/reference/ft2-pfr_fonts.html
index 19e3dd4..8af3974 100644
--- a/docs/reference/ft2-pfr_fonts.html
+++ b/docs/reference/ft2-pfr_fonts.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-properties.html b/docs/reference/ft2-properties.html
index 8b0f755..77adb40 100644
--- a/docs/reference/ft2-properties.html
+++ b/docs/reference/ft2-properties.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-quick_advance.html b/docs/reference/ft2-quick_advance.html
index 3c03365..94da948 100644
--- a/docs/reference/ft2-quick_advance.html
+++ b/docs/reference/ft2-quick_advance.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-raster.html b/docs/reference/ft2-raster.html
index 674e8ea..3aa176b 100644
--- a/docs/reference/ft2-raster.html
+++ b/docs/reference/ft2-raster.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-sfnt_names.html b/docs/reference/ft2-sfnt_names.html
index 34a78ee..d32bc2c 100644
--- a/docs/reference/ft2-sfnt_names.html
+++ b/docs/reference/ft2-sfnt_names.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-sizes_management.html b/docs/reference/ft2-sizes_management.html
index afb7026..cafbcef 100644
--- a/docs/reference/ft2-sizes_management.html
+++ b/docs/reference/ft2-sizes_management.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-system_interface.html b/docs/reference/ft2-system_interface.html
index 6f65986..64c148f 100644
--- a/docs/reference/ft2-system_interface.html
+++ b/docs/reference/ft2-system_interface.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-t1_cid_driver.html b/docs/reference/ft2-t1_cid_driver.html
index 6f8a9b9..5c8dd55 100644
--- a/docs/reference/ft2-t1_cid_driver.html
+++ b/docs/reference/ft2-t1_cid_driver.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-truetype_engine.html b/docs/reference/ft2-truetype_engine.html
index 770178d..21dafaf 100644
--- a/docs/reference/ft2-truetype_engine.html
+++ b/docs/reference/ft2-truetype_engine.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-truetype_tables.html b/docs/reference/ft2-truetype_tables.html
index adced45..17bd853 100644
--- a/docs/reference/ft2-truetype_tables.html
+++ b/docs/reference/ft2-truetype_tables.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-tt_driver.html b/docs/reference/ft2-tt_driver.html
index 5a9611e..372887e 100644
--- a/docs/reference/ft2-tt_driver.html
+++ b/docs/reference/ft2-tt_driver.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-type1_tables.html b/docs/reference/ft2-type1_tables.html
index 4166da3..b3927d1 100644
--- a/docs/reference/ft2-type1_tables.html
+++ b/docs/reference/ft2-type1_tables.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-user_allocation.html b/docs/reference/ft2-user_allocation.html
index 7a71678..5ea22ee 100644
--- a/docs/reference/ft2-user_allocation.html
+++ b/docs/reference/ft2-user_allocation.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-version.html b/docs/reference/ft2-version.html
index f4b291b..bac8584 100644
--- a/docs/reference/ft2-version.html
+++ b/docs/reference/ft2-version.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/ft2-winfnt_fonts.html b/docs/reference/ft2-winfnt_fonts.html
index ff81b5b..d8eb82d 100644
--- a/docs/reference/ft2-winfnt_fonts.html
+++ b/docs/reference/ft2-winfnt_fonts.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
diff --git a/docs/reference/index.html b/docs/reference/index.html
index 021146f..12b0770 100644
--- a/docs/reference/index.html
+++ b/docs/reference/index.html
@@ -59,8 +59,6 @@
     
     
       
-        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Serif:300,400,400i,700%7CRoboto+Mono&display=fallback">
         <style>body,input{font-family:"Noto Serif","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style>
       
     
